sdk/evaluation/azure-ai-evaluation/CHANGELOG.md

```diff
@@ -66,6 +66,26 @@ evaluate(
 )
 ```
 
+- Simulator now requires configuration to call the prompty within the `azure_ai_project`. This enables the usage of simulator with Entra ID based auth.
```

Comment:
```suggestion
- Simulator now requires a model configuration configuration to call the prompty instead of an Azure AI project scope. This enables the usage of simulator with Entra ID based auth.
```